src/Headers/DomainBox.h

```diff
@@ -75,12 +75,12 @@ MPI_Datatype CreateBoxType (Box<ndim> dummy) {
 /// \date   03/04/2013
 //=================================================================================================
 template <int ndim>
-struct DomainBox : public Box<3> {
-  boundaryEnum boundary_lhs[3];     ///< LHS boundary types
-  boundaryEnum boundary_rhs[3];     ///< RHS boundary types
-  FLOAT size[3];                    ///< Side-lengths of bounding box
-  FLOAT half[3];                    ///< Half side-lengths of bounding box
-  bool PeriodicGravity ;               ///< Whether the domain is using periodic gravity.
```

Comment:
Yes, I was confused why this was hard-coded into 3d in the first place.  The Ewald object is only ever constructed in 3d (and throws an exception if we attempt to use it in 1d/2d) so I'm not sure if this was needed.  If there's a reason to keep it that I've not thought about, we can but otherwise we can get rid of it later.